security-framework-sys/src/access_control.rs

```diff
@@ -0,0 +1,25 @@
+use core_foundation_sys::base::{CFAllocatorRef, CFTypeRef};
+use core_foundation_sys::error::CFErrorRef;
+use core_foundation_sys::base::CFOptionFlags;
+
+use crate::base::SecAccessControlRef;
+
+mod access_control_flags {
```

Comment:
if you have `pub use` right after it, then the module seems redundant.

Reply:
oh ok thanks, I was reusing the syntax I saw in another file

Reply:
actually i'm not sure how to address this